nixos/modules/services/editors/emacs.nix

```diff
@@ -70,8 +69,7 @@ in
       description = "Emacs: the extensible, self-documenting text editor";
 
       serviceConfig = {
-        Type = "forking";
```

Comment:
Upstream uses `Type=notify`.  Could you also change that?

Reply:
Done